packages/apps/shopify-app-remix/src/server/types-contexts.ts

```diff
@@ -39,6 +43,9 @@ export type AdminContext<App = DefaultApp> = IAdminContext<
   ConfigComponents<ShopifyConfig<App>>['resources']
 >;
 
+export type ScopesContext<_App = DefaultApp> = IScopesContext;
+export type ScopesDetail = IScopesDetail;
```

Comment:
Nice! Good to make sure we're exposing these type helpers out of the package too.

Reply:
The goal of exporting these types here is to make it possible for apps to create functions that take those, like:

```ts
function myFn(scopes: ScopesContext<typeof shopify>) {
  scopes.revoke(...);
}
```

and have TS sort out the types correctly from that `shopify` instance's configs.